validators.js

```diff
@@ -0,0 +1,29 @@
+const checkEmail = (email) => {
+  const re = new RegExp(
```

Comment:
re라는 변수명을 통해 어떤 값이 할당되었는지 예측할 수 없습니다.

변수를 선언한 이후 실제 `re`라는 변수를 사용할 때 혹은 다른 동료 개발자가 해당 변수를 보았을 떄 그 의미가 명확하게 전달될 수 있도록 수정해주세요. (해당 부분은 다음부터 꼼꼼하게 확인해주세요.)

해당 부분은 이메일 형식의 유효성검사를 위한 정규표현식이 할당되기 떄문에 다음과 같이 설정할 수 있습니다.

변수를 어떻게 정해야 할 지 모르겠다면 아래의 방법을 참고해주세요.

1. 해당 변수 혹은 함수가 어떤 행위를 하고 어떤 결과를 보이는지 파악하기
2. 팀에서 변수명을 지을 때 어떻게 짓는지 팀 컨벤션 확인하기
3. 팀원에게 도움 요청하기

```suggestion
  const emailRegex = new RegExp(
```